module/zfs/qat_crypt.c

```diff
@@ -72,11 +78,23 @@ symcallback(void *p_callback, CpaStatus status, const CpaCySymOp operation,
 	}
 }
 
+static void
+symsessionwait(CpaCySymSessionCtx cy_session_ctx)
+{
+/* Session reuse is available since Cryptographic API version 2.2 */
+#if CY_API_VERSION_AT_LEAST(2, 2)
+	CpaBoolean inuse = CPA_FALSE;
+	do {
+		cpaCySymSessionInUse(cy_session_ctx, &inuse);
```

Comment:
Should we be conditionally rescheduling in this loop to prevent a potential tight spin?